src/main/groovy/life/qbic/portal/sampletracking/trackinginformation/update/UpdateSampleTrackingInfo.groovy

```diff
@@ -35,12 +29,52 @@ class UpdateSampleTrackingInfo implements SampleTrackingUpdateInput {
     @Override
     def setCurrentSampleLocation(String sampleId, Location location) {
         try {
-            this.sampleTrackingUpdateDataSource.updateSampleLocation(sampleId, location)
+            updateSampleLocation(sampleId, location)
```

Comment:
why this fix here?

Reply:
Because you call it also with the new introduced method?

Reply:
I introduced a local method that handles the call because I need the same behaviour for the newly introduced API method.

Reply:
I could also have left the direct call to the data source here 👀

Reply:
Yes, I think so too. But its fine for me if you want to keep it as is

Reply:
I will keep it then 😄